core/src/main/java/io/questdb/log/LogAlertSocket.java

```diff
@@ -304,6 +308,69 @@ int getDefaultAlertPort() {
         return defaultPort;
     }
 
+
+    private static boolean isContentLength(CharSequence message, int lo, int hi) {
+        int len = hi - lo;
+        if (len < CONTENT_LENGTH.length) {
+            return false;
+        }
+        for (int i = 0; i < CONTENT_LENGTH.length; i++) {
```

Comment:
There is faster and better way to check for constants in CharSequence. Please have a look at `SqlKeywords`:

```java
    public static boolean isAlterKeyword(CharSequence tok) {
        if (tok.length() != 5) {
            return false;
        }

        int i = 0;
        return (tok.charAt(i++) | 32) == 'a'
                && (tok.charAt(i++) | 32) == 'l'
                && (tok.charAt(i++) | 32) == 't'
                && (tok.charAt(i++) | 32) == 'e'
                && (tok.charAt(i) | 32) == 'r';
    }

```

Reply:
yeah, that is where I took inspiration from, my solution does not unroll, I considered it but it was far less readable. I will change it.